### PR TITLE
feat(insights): group actor modals

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

Turns out actor group queries for HogQL insights were broken or not implemented.

## Changes

- Some improvements to the HogQL comparison script
- Change the data tables to always say "1 entry / 2 entries", as the "1 persons" detection was getting complicated (for stickiness group results, I'd need to look inside each of the query's series to determine what's what)
- Make group actors support work for stickiness
- Make group actors support work for trends
- Improve display of group results for retention queries
- Make it possible to edit columns in the table for the retention groups query
- Show returned properties or other object-as-string in a nice JSON view in the data table

Trends:

Stickiness:

Retention:

## How did you test this code?

- Added a test for the stickiness actors query and the stickiness group actors queries.
- Added a test for trends actor and group actor queries

### CODE REVIEW

**Review by Gilbert09 — APPROVED** [review]

File: posthog/hogql_queries/actors_query_runner.py [hunk 1]
```diff
@@ -123,6 +123,10 @@ def source_id_column(self, source_query: ast.SelectQuery | ast.SelectUnionQuery)
         else:
             select = source_query.select_queries[0].select
 
+        for column in select:
+            if isinstance(column, ast.Alias) and (column.alias in ("group_key", "actor_id", "person_id")):
```
Comment (webjunkie) [inline posthog/hogql_queries/actors_query_runner.py:127]: Why not move this into the loop right below 🙈
It's basically the condition that is there just modified.

File: posthog/hogql_queries/actors_query_runner.py [hunk 2]
```diff
@@ -163,7 +167,7 @@ def to_query(self) -> ast.SelectQuery:
 
                 if expr == "person.$delete":
                     column = ast.Constant(value=1)
-                elif expr == self.strategy.field:
+                elif expr == self.strategy.field or expr == "actor":
```
Comment (webjunkie) [inline posthog/hogql_queries/actors_query_runner.py:170]: `elif expr in (self.strategy.field, "actor"):` might read better

**Review by webjunkie — COMMENTED** [review]
Just a couple of post-merge notes.

And I had a small reason why I used person/group vs. just actor in the select query... but cannot remember right now 🤷🏻

File: posthog/hogql_queries/actors_query_runner.py (hunk 1 above)
Comment (mariusandra) [reply posthog/hogql_queries/actors_query_runner.py:127]: Mostly for precedence. Otherwise "session_id" or something may end up first and cause mayhem

**Pull request merged**